liberty-maven-plugin/src/it/tests/appsdirectory-set-appz-notconfigured-it/src/test/java/net/wasdev/wlp/maven/test/app/DefaultAppDirectoryIT.java

```diff
@@ -0,0 +1,21 @@
+package net.wasdev.wlp.maven.test.app;
+
+import java.io.File;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+/**
+ * 
+ * Web application test case
+ * 
+ */
+
+public class DefaultAppDirectoryIT {
+    
+    @Test
+    public void testApplicationFileExist() throws Exception {
+        File f = new File("target/liberty/usr/servers/test/dropins/appsdirectory-set-appz-configured-it.war");
```

Comment:
should be "*notconfigured-it.war"  Double check the folder because not sure how this passed.